docs/main_text.md

```diff
@@ -224,10 +225,7 @@ This is also known as an inner join and is illustrated below.
 ![inner join](https://user-images.githubusercontent.com/15315514/44534751-7a46dd00-a6c6-11e8-9760-e8daa91a500f.png)
 Note that some early generation microarrays measure fewer genes than their more recent counterparts, so their inclusion when aggregating `by species` may result in a small number of genes being returned. 
 
-
-The aggregation methodology for species compendia is different; see [Species compendia](#species-compendia) for more information.
-
-TODO: illustration of structure of download zip file
+The aggregation methodology for future releases of species compendia will be different; see [Species compendia](#species-compendia) for more information.
```

Comment:
Doesn't this imply that refine.bio currently has a species compendia available?

Reply:
Do you think I should remove this from this section entirely? That's what I'm leaning towards.